human_robot_handover_ros:
    ggcnn: 
        topic: /ggcnn_humanseg
        window: 5
        deviation_position: 0.07
        deviation_orientation: 0.2
    robot:
        arm_state: /arm/state
        arm_servo_pose: /arm/cartesian/servo_pose
        arm_named_pose: /arm/cartesian/named_pose
        arm_gripper: /arm/gripper
    movement:
        position_error: 0.03
        move_x: True
        move_y: True
        move_z: True
        dist_ggcnn: 0.5
        dist_final: 0.2
        move_final: True
        speed_approach: 0.4
        scaling_handover: 1
    gripper:
        gripper_open: 0.08
    visualization: 
        topic: /handover/velocity
        activated: True